spyder/plugins/editor/plugin.py

```diff
@@ -1653,13 +1653,13 @@ def update_recent_file_menu(self):
         """Update recent file menu"""
         recent_files = []
         for fname in self.recent_files:
-            if self.is_file_opened(fname) is None and osp.isfile(fname):
+            if osp.isfile(fname):

```

Comment:
@juanis2112 question, why did we remove `self.is_file_opened(fname)`?

Reply:
@goanpeca I understand this line is the one that doesn't let the entry to be created while the file is open. Please correct me if I'm wrong.

Reply:
Yes you are right, missed that part, I thought this was only about the extension!, y bad :-)